test/jdk/java/awt/TrayIcon/TrayIconScalingTest.java

```diff
@@ -51,7 +51,6 @@ public class TrayIconScalingTest {
 
     private static SystemTray tray;
     private static TrayIcon icon;
-
```

Comment:
I'm nitpicking but I guess the blank line between the fields used in the test and instructions was in its own place to separate the two different groups.